spec/graphql/types/query_type_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Types::QueryType" do
+  describe "Listings" do
+    let!(:listings) { create_pair(:listing) }
+
+    let(:query) do
+      %(
+        query{
+          listings{
+            title
+          }
+        }
+      )
+    end
+
+    subject(:result) do
+      NeighbourlySchema.execute(query).as_json
```

Comment:
call the Graphql::Schema.execute() to mock the server processing and responding to our query. save that value as `result` and make that the "subject" of the test